### PR TITLE
Fixes GitHub actions OOM error.

### PR DESCRIPTION
**Pull request by esirK**
## Description
Fixes GitHub actions OOM error.
This issue looks similar to [this](https://github.com/SpiderLabs/ModSecurity/issues/2381) 
Fixes # (issue)
The fix uses Nginx alpine version which is both smaller in size and has the memory leak bug fixed.
Given that we only use `docker-compose` for local development, I think it's alright for us to `alpine` image in our compose file.
## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: .github/workflows/pull_request.yml [hunk 1]
```diff
@@ -20,6 +20,7 @@ jobs:
       - name: Build the Docker image
         run: |
           echo "ACTNOW_SECRET_KEY=${{ secrets.ACTNOW_SECRET_KEY }}" >> .env
+          echo "ACTNOW_SENTRY_DSN=${{ secrets.ACTNOW_SENTRY_DSN }}" >> .env
```
Comment (kilemensi) [inline .github/workflows/pull_request.yml]: `SECRET_KEY` is required while `SENTRY_DSN` is optional. Why this change?

File: docker-compose.yml [hunk 2]
```diff
@@ -34,7 +34,7 @@ services:
     stdin_open: true
     tty: true
   nginx:
-    image: nginx:1.19
+    image: nginx:1.19.10-alpine
```
Comment (kilemensi) [inline docker-compose.yml]: We use `ubuntu` / `debian` in prod, any chance of using either of them here?

**Review by kilemensi — COMMENTED** [review]
👍🏾 

---

PR says remove unless-stopped option but I don't see where you're removing it @esirK .

**Comment by esirK** [comment]
> PR says remove unless-stopped option but I don't see where you're removing it @esirK .

Ignore that. I've updated the title.

File: .github/workflows/pull_request.yml (hunk 1 above)
Comment (esirK) [reply .github/workflows/pull_request.yml]: It was raising an exception because of not being set. I've removed this part and updated the code to make it optional.

File: docker-compose.yml (hunk 2 above)
Comment (esirK) [reply docker-compose.yml]: > We use ubuntu / debian in prod, any chance of using either of them here?

Yep. Although it's much bigger, I've used `nginx:1.20` which uses `Debian` as the base image.

**Comment by zimmerle** [comment]
@esirK :)

**Comment by esirK** [comment]
> @esirK :)

Hey @zimmerle
